build/teamcity/internal/release/build-and-publish-patched-go/impl-fips.sh

```diff
@@ -10,7 +10,7 @@ set -xeuo pipefail
 
 GO_FIPS_REPO=https://github.com/golang-fips/go
 GO_FIPS_COMMIT=fc8a2bd706bcd6b18b260a9aea59cf63884acdeb
-GO_VERSION=1.22.8
+GOCOMMIT=$(cat /bootstrap/commit.txt)
```

Comment:
Maybe do something like `grep -v ^# /bootstrap/commit.txt | head -n1` to allow comments in `commit.txt`?